tests/test_pe.py

```diff
@@ -182,6 +182,46 @@ def test_pe_win_x8664_fls(self):
         del ql
 
 
+    def test_pe_win_x86_return_from_main_stackpointer(self):
+        """
+            Sample binary compiled with tcc: `tcc return_main.c -nostdlib -o return_main.exe`
+return_main.c:
+    int _start()
```

Comment:
Sorry I'm still not so clear whether this program behaves differently on win32 and posix systems?